### ISSUE TITLE
denali command errors

### ISSUE THREAD

**Issue by knownasilya** [denali-js/denali#383]
```
▲ ~/sandbox/denali-rethinkdb denali help g                                                                         master :: 2m :: ⬡
cli v0.0.14 [local] Error encountered while starting up denali-cli:
AssertionError [ERR_ASSERTION]: Denali package was found, but unable to load core commands - is your Denali installation corrupted?
    at addons.forEach (/Users/iradchenko/sandbox/denali-rethinkdb/node_modules/denali-cli/lib/bootstrap.ts:57:7)
    at Array.forEach (<anonymous>)
    at Object.run [as default] (/Users/iradchenko/sandbox/denali-rethinkdb/node_modules/denali-cli/lib/bootstrap.ts:52:10)
    at Object.<anonymous> (/usr/local/lib/node_modules/denali-cli/bin/denali.ts:58:74)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
    at Function.Module._load (module.js:462:3)
    at Function.Module.runMain (module.js:609:10)
```

Pretty much any command. I'm using Node 8.4.
Using global latest and local latest versions.

Build gives me this

```
denali build

denali-cli 0.0.14 [local]
Error encountered while starting up denali-cli:
Error: Cannot find module '/Users/iradchenko/sandbox/denali-rethinkdb/node_modules/denali-cli/dist/bootstrap'
    at Function.Module._resolveFilename (module.js:489:15)
    at Function.Module._load (module.js:439:25)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/iradchenko/sandbox/denali-rethinkdb/node_modules/denali-babel/node_modules/denali-cli/src/bin/denali.ts:54:7)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
    at Function.Module._load (module.js:462:3)
```

**Comment by davewasmer** [comment]
Hmm I remember seeing this recently, I don't recall exactly the problem though.

I couple things to look into:

1. Is denali-babel using an old version of the cli or core? In other words - you might have updated Denali-rethinkdb to use latest both, but perhaps another dep in your node_modules folder has them at al older version?

2. The "is Denali corrupted?" error happens when we find the Denali package, but can't actually load the commands from it. Try running with DEBUG=denali* to get some more details. If you do t see any obvious problems, try looking at where the debug output claims to be finding the Denali package - does that look right? Are there commands there?

I've made a lot of recent improvements to the debug output which should help diagnosing some of the build issues, so that's a good first stop in figuring out the problem.

**Comment by knownasilya** [comment]
You are correct on the cli being different versions:

```
denali-rethinkdb@0.2.4 /Users/iradchenko/sandbox/denali-rethinkdb
├─┬ denali@0.0.32
│ └── denali-cli@0.0.12
├─┬ denali-babel@0.0.13
│ └── denali-cli@0.0.3
├── denali-cli@0.0.14
└─┬ denali-eslint@0.0.14
  └── denali-cli@0.0.12
```

**Comment by knownasilya** [comment]
Looks like denali itself uses an older version too

**Comment by seawatts** [comment]
Pretty sure this can be closed now. Fixed in v0.0.37
